utils/time_test.go

```diff
@@ -15,36 +17,30 @@ func TestMillisFromTime(t *testing.T) {
 	actual := MillisFromTime(input)
 	expected := int64(1420115640000)
 
-	if actual != expected {
-		t.Fatalf("TestMillisFromTime failed, %v=%v", expected, actual)
-	}
+	assert.Equal(t, actual, expected)
```

Comment:
Can we reverse the order of the parameters here and in the rest of the file? After the `*testing.T` parameter, it should be `expected` and then `actual` -- see https://godoc.org/github.com/stretchr/testify/assert#Equal.

This matters since the error message generated by testify will say it "expected" one thing but "got actual", which might be confusing.

Reply:
Sure, sorry about this mistake.